ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -1184,30 +1184,29 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )okd-e2e-gcp-op,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
     branches:
     - master
     cluster: api.ci
-    context: ci/prow/okd-e2e-gcp-upgrade
+    context: ci/prow/okd-e2e-upgrade
     decorate: true
     decoration_config:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
       ci-operator.openshift.io/variant: okd
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-machine-config-operator-master-okd-e2e-gcp-upgrade
-    optional: true
```

Comment:
Shouldn't we keep this job optional? now that it will be running on all PRs , on failure it will block PRs from getting merged.

Reply:
It was renamed into `e2e-upgrade`

Reply:
From the diff, it looks to me that this change is about okd-e2e-upgrade test

Reply:
:man_facepalming: I misread this, sorry. Added missing `optional` and updated `always_run`